src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/cosmic/CosmicFuncotationFactory.java

```diff
@@ -358,15 +373,24 @@ private final SimpleInterval getGenomePositionFromResults(final ResultSet result
      * @param resultSet The results of a query on the database with a current row (must not be {@code null}).
      * @return A {@link SimpleInterval} representing the extents of the protein position in the current record in the given {@link ResultSet} or {@code null}.
      */
-    private final SimpleInterval getProteinPositionFromResults(final ResultSet resultSet) {
+    private SimpleInterval getProteinPositionFromResults(final ResultSet resultSet) {
+        return parseProteinString( getProteinChangeStringFromResults(resultSet) );
+    }
+
+    /**
+     * Pulls a protein change string out of the current record in the given {@link ResultSet}.
+     * @param resultSet The results of a query on the database with a current row (must not be {@code null}).
+     * @return A {@link String} representing the protein change as found in the current record of the given {@link ResultSet}.  Will not be {@code null}.
+     */
+    private String getProteinChangeStringFromResults(final ResultSet resultSet) {
         Utils.nonNull(resultSet);
 
         try {
-            final String rawPosition = resultSet.getString(PROTEIN_POSITION_COLUMN_NAME);
-            return parseProteinString(rawPosition);
+            final String proteinChangeString = resultSet.getString(PROTEIN_POSITION_COLUMN_NAME);
+            return proteinChangeString == null ? "" : proteinChangeString;
         }
         catch (final SQLException ex) {
-            throw new GATKException("Cannot get Protein Position from column: " + GENOME_POSITION_COLUMN_NAME, ex);
+            throw new GATKException("Cannot get Protein Change from column: " + GENOME_POSITION_COLUMN_NAME, ex);
```

Comment:
I think protein change should be lower case

Reply:
Fixed!